src/Promitor.Core.Scraping/Configuration/Serialization/v1/Core/MetricDimensionDeserializer.cs

```diff
@@ -1,24 +1,14 @@
 ﻿using Microsoft.Extensions.Logging;
 using Promitor.Core.Scraping.Configuration.Serialization.v1.Model;
-using YamlDotNet.RepresentationModel;
 
 namespace Promitor.Core.Scraping.Configuration.Serialization.v1.Core
 {
     public class MetricDimensionDeserializer : Deserializer<MetricDimensionV1>
     {
-        private const string NameTag = "name";
-
         public MetricDimensionDeserializer(ILogger<MetricDimensionDeserializer> logger)
             : base(logger)
         {
-        }
-
-        public override MetricDimensionV1 Deserialize(YamlMappingNode node, IErrorReporter errorReporter)
```

Comment:
Don't we need this anymore?

Reply:
No - it looked like this mapper just hadn't been updated to use the new mapping model - it was doing it manually by overriding Deserialize. Since it's just getting the name field and not doing anything complicated, I just converted it while I was going through all the files.

Reply:
Sounds good to me, can we just verify that we have a test covering this change please?

Reply:
Yeah - here you go: https://github.com/tomkerkhove/promitor/blob/master/src/Promitor.Tests.Unit/Serialization/v1/Core/MetricDimensionDeserializerTests.cs#L19

It was written by your very own hand! :smile_cat:

Reply:
Perfect :D

If I wrote it, then I don't trust it :D